vendor/github.com/orisano/fls/endian_big.go

```diff
@@ -0,0 +1,34 @@
+// +build ppc64 s390x mips mips64
+
+// Copyright 2016 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions are
+// met:
+//
+//    * Redistributions of source code must retain the above copyright
+// notice, this list of conditions and the following disclaimer.
+//    * Redistributions in binary form must reproduce the above
```

Comment:
I don't think you need to repaste the license into each file.  Having it as the license is good enough

Reply:
sorry, I don't know the right way of redistributing the licensed software.